Formula/m/msolve.rb

```diff
@@ -4,6 +4,7 @@ class Msolve < Formula
   url "https://github.com/algebraic-solving/msolve/archive/refs/tags/v0.9.0.tar.gz"
   sha256 "742e84cf4d11eeadf62002623ecb7658e5d6d8c838fcf571fac06acf44252983"
   license "GPL-2.0-or-later"
+  revision 1
```

Comment:
Most users build from bottles, and therefore we should not do a revision bump (because it will force most users to reinstall the exact same version).